docs/PRD-linear-5d882b0c-d553-4d63-8d62-6f973090a13c.md

```
@@ -0,0 +1,129 @@
+  - `codex-orchestrator flow --task <task-id>`
+  - `codex-orchestrator doctor --format json`
+- Nearby wrong interpretations to reject:
+  - replacing the README with marketing copy
+  - deleting detailed public guidance instead of moving it
+  - promoting `0.124.0` merely because it is installed locally
+  - claiming the local auto-continue hook governs child/provider sessions without checking its state and predicates
+  - changing runtime/model defaults as part of a docs cleanup
+  - editing the dirty shared checkout
+
+## Parity / Alignment Matrix
+| Surface | Current Truth | Reference Truth | Target Truth | Explicitly Out Of Scope |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)